brax/envs/wrappers/training.py

```diff
@@ -46,11 +46,11 @@ def wrap(
     environment did not already have batch dimensions, it is additional Vmap
     wrapped.
   """
-  env = EpisodeWrapper(env, episode_length, action_repeat)
   if randomization_fn is None:
     env = VmapWrapper(env)
   else:
     env = DomainRandomizationVmapWrapper(env, randomization_fn)
+  env = EpisodeWrapper(env, episode_length, action_repeat)
```

Comment:
just OOC why is this move needed?

Reply:
Something about how we implemented the batching rules that we use to call Madrona means that `vmap(scan(render()))` works but `scan(vmap(render()))` fails.

When we have some time we will fix this in `madrona_mjx` but for now it's easier to fix it here.

Reply:
The Madrona Batch Renderer breaks when you do vmap(scan(env.step)) but is ok for scan(vmap(env.step)).

Reply:
This change is no longer needed, since Madrona requires it's own vmap wrapper to be used:
https://github.com/shacklettbp/madrona_mjx/blob/main/src/madrona_mjx/wrapper.py

wrap_env must be False when using vision, so that users know to use the MadronaWrapper

Reply:
Nice.  I would be OK with merging this as is and then adopting the Madrona wrapper as a separate PR.